src/main/java/picard/illumina/parser/TileMetricsUtil.java

```diff
@@ -111,13 +117,33 @@ public static File renderTileMetricsFileFromBasecallingDirectory(final File illu
                 final int phasingCode = IlluminaMetricsCode.getPhasingCode(descriptorIndex, IlluminaMetricsCode.PHASING_BASE);
                 final int prePhasingCode = IlluminaMetricsCode.getPhasingCode(descriptorIndex, IlluminaMetricsCode.PREPHASING_BASE);
 
-                if (!(codeMetricsMap.containsKey(phasingCode) && codeMetricsMap.containsKey(prePhasingCode))) {
-                    throw new PicardException("Don't have both phasing and prephasing values for tile");
+                float phasingValue = 0;
+                float prePhasingValue = 0;
+
+                // If both the phasing and pre-phasing data are missing, then likely something went wrong when imaging
+                // this tile, for example a grain of sand disrupting the path of light to the sensor.  If only one of them
+                // is missing, then likely the data is corrupt.
+                if (!codeMetricsMap.containsKey(phasingCode) && !codeMetricsMap.containsKey(prePhasingCode)) {
+                    final String message = String.format(
+                            "Don't have both phasing and prephasing values for %s read cycle %s.  Phasing code was %d and prephasing code was %d.",
+                            tileTemplateRead.toString(), descriptorIndex + 1, phasingCode, prePhasingCode
+                    );
+                    switch (validationStringency) {
+                        case LENIENT:
+                            LOG.warn(message);
+                            // NB: intentionally no break
```

Comment:
why no break? I mean, sure it will break due to the next case, but why code this way? seems brittle.